src/EditorFeatures/CSharpTest/Diagnostics/AddUsing/AddUsingTests_NuGet.cs

```diff
@@ -0,0 +1,241 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.CodeFixes.AddImport;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Packaging;
+using Moq;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics.AddUsing
+{
+    using System.Collections.Immutable;
+    using FixProviderData = Tuple<IPackageInstallerService, IPackageSearchService>;
+
+    public partial class AddUsingTests
+    {
+        const string NugetOrgSource = "nuget.org";
+
+        public class NuGet : AddUsingTests
+        {
+            private static readonly ImmutableArray<string> NugetPackageSources =
+                ImmutableArray.Create(NugetOrgSource);
+
+            internal override Tuple<DiagnosticAnalyzer, CodeFixProvider> CreateDiagnosticProviderAndFixer(
+                Workspace workspace, object fixProviderData)
+            {
+                var data = (FixProviderData)fixProviderData;
+                return Tuple.Create<DiagnosticAnalyzer, CodeFixProvider>(
+                        null,
+                        new CSharpAddImportCodeFixProvider(data.Item1, data.Item2));
+            }
+
+            protected override IList<CodeAction> MassageActions(IList<CodeAction> actions)
+            {
+                return FlattenActions(actions);
+            }
+
+            [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddUsing)]
+            public async Task TestSearchPackageSingleName()
+            {
+                // Make a loose mock for the installer service.  We don't care what this test
+                // calls on it.
+                var installerServiceMock = new Mock<IPackageInstallerService>(MockBehavior.Loose);
+                installerServiceMock.SetupGet(i => i.PackageSources).Returns(NugetPackageSources);
+
+                var packageServiceMock = new Mock<IPackageSearchService>();
```

Comment:
In these tests, you have a descriptive comment on mock behaviour for the `installerServiceMock`, but neither a comment nor a `Verify()` call for the `packageServiceMock`